fidimag/micro/llg.py

```diff
@@ -10,6 +10,7 @@
 from fidimag.common.integrators import SundialsIntegrator, StepIntegrator
 import fidimag.common.constant as const
 import fidimag.common.helper as helper
+import fidimag.common.skyrmion_number
```

Comment:
I think we should create a LLG super class or sim_tools in the future, as we discussed, so we avoid importing everything we add in the LLG class